decidim-admin/spec/system/admin_manages_officializations_spec.rb

```diff
@@ -81,7 +81,7 @@
           :user,
           :officialized,
           officialized_as: { "en" => "Mayor of Barcelona" },
-          organization: organization
+          organization:    organization
```

Comment:
I'm not sure this is our current style, but I think it's not. Can you confirm?

Reply:
It's not, wrong configuration of rubymine.